src/components/registration/registration.jsx

```diff
@@ -36,18 +34,14 @@ var Registration = React.createClass({
         this.toggleMessageListener(false);
     },
     render: function () {
-        var frameProps = {
-            width: 610,
-            height: 438
-        };
         return (
-            <Modal
-                    isOpen={this.props.isOpen}
-                    onRequestClose={this.props.onRequestClose}
-                    className="registration"
-                    style={{content:frameProps}}>
-                <iframe ref="registrationIframe" src="/accounts/standalone-registration/" {...frameProps} />
-            </Modal>
+            <IframeModal
+                isOpen={this.props.isOpen}
+                onRequestClose={this.props.onRequestClose}
+                className="mod-registration"
+                componentRef="registrationIframe"
```

Comment:
I think this will break the end of the registration flow since onMessage refers to this.refs.registrationIframe which won't exist anymore. This is the perfect place for a function ref, like `iframeRef={function(iframe){ this.registrationIframe = iframe; }}`, and then in `onMessage`, `this.registrationIframe.contentWindow.location.reload()`.